app/src/main/res/drawable/top_sites_background.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<shape
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:shape="rectangle">
+    <corners android:radius="4dp" />
+    <solid
+        android:color="?mozac_widget_favicon_background_color"
+        tools:color="@color/photonWhite" />
+    <stroke
+        android:width="1dp"
+        android:color="@color/top_sites_border_normal_theme"
```

Comment:
I think we should just call this `top_sites_border_color`. I think this is just reinforcing the confusion we already have. See `top_site_pin_icon_background_tint`. Realistically, you just need one variable name in each of the colors.xml and it will be mapped accordingly.